### DISCUSSION TITLE
Should `rec_control reload-acls` also reload `proxy-protocol-from`?

### DISCUSSION THREAD

**Discussion by franklouwers** [PowerDNS/pdns#14086] [Q&A]
I was looking for a way to reload pdns_rec after a change has been made to `proxy-protocol-from`. I noticed `rec_control reload-acls` and was wondering if it should reload `proxy-protocol-from`? 

At present it doesn't, however the internal variable to track proxy-protocol-from is called `g_proxyProtocolACL`, suggesting the devs at one point felt it's an ACL.

If it's an ACL, should it be reloaded bu `reload-acls`?

**Comment by omoerbeek** [comment]
I think that would make sense. I'll check if it is feasible soon.

**Reply by omoerbeek** [reply]
It is feasible, needs a bit of work. I'm moving this to an issue.
